docs/builders/vsphere-clone.mdx

```diff
@@ -45,6 +45,24 @@ necessary for this build to succeed and can be found further down the page.
 
 @include 'builder/vsphere/clone/CloneConfig-not-required.mdx'
 
+Example of usage:
+
+<Tabs>
+<Tab heading="JSON">
+```json
+"network": "new_network_for_network_adapter_0, , new_network_for_network_adapter_2",
```

Comment:
Did you really want 2 commas in these examples?

Reply:
Yes, absolutely.
As the title says for this example: Reconfiguring only some network adapters

Reply:
OK got it, you just want to configure 0 & 2, not 1, hence the 2 commas

Reply:
I included the examples given in the pull request description into the help file.
This should explain it well.
But, how can I get the ammended comit in my fork now into this pull request....